standard/sections/clause_7_monitoring_event_topic.adoc

```diff
@@ -12,18 +12,15 @@ include::../requirements/requirements_class_monitoring-event-topic.adoc[]
 
 Successful operation of the WIS2 infrastructure and monitoring events of same should be information that is made available to all Global Services and WIS2 Nodes, and not designed for communication to external users or data consumers.  Global Services need to be able to report information to Global Services and WIS2 Nodes to trigger corrective action.
 
-The WMET is composed of five levels: A fixed channel of `monitor`, WTH primary topic levels 2 (version) and 3 (system), as well as WTH primary topic level 4 (centre identifier) at two WMET levels:
-
-* _producer_: the centre identifier of the entity producing the event notification
-* _subject: the centre identifier of the subject of the event notification
+The WMET is composed of four levels: A fixed channel of `monitor`, WTH primary topic levels 2 (version), 3 (system), and 4 (centre identifier).  Level 4 is thethe centre identifier of the entity producing the event notification (the _producer_).
```

Comment:
The WMET is composed of four levels: A fixed channel of `monitor`, WTH primary topic levels 2 (version), 3 (system), and 4 (centre identifier). Level 4 is the centre identifier of the entity subject of the event notification (the _subject_).